Unix/cloc

```diff
@@ -8829,6 +8829,10 @@ sub normalize_file_names {                   # {{{1
                 $F_norm = lc "$cwd/$F_norm";
             }
         }
+        # Remove trailing / so it does not interfere with further regex code 
+        # that does not expect it
+        $F_norm =~ s{/$}{}g;
```

Comment:
yes, I've supposed only correct directory names.

sorry, for git commit messages. I've just tried to do it through github UI and missed where it can be set.